### PR TITLE
Easy opt-out link

### PR DESCRIPTION
**Pull request by gsvr**
Add opt-out link to private note

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: snappybouncer/tasks.py [hunk 1]
```diff
@@ -112,8 +113,15 @@ def update_snappy_ticket_with_extras(snappy_api, nonce, contact_key, subject):
     contact = contacts_api.get_contact(contact_key)
     extra_info = ""
     for extra in settings.SNAPPY_EXTRAS:
+        # Add available contact extras
         if extra in contact["extra"]:
             extra_info += extra + ": " + contact["extra"][extra] + "\n"
+        # Add opt-out link
+        if extra == "controlinterface_optout_link":
```
Comment (imsickofmaps) [inline snappybouncer/tasks.py]: Bring this in from the extras for loop please as discussed.

File: snappybouncer/tasks.py [hunk 2]
```diff
@@ -112,19 +112,24 @@ def update_snappy_ticket_with_extras(snappy_api, nonce, contact_key, subject):
...
-                "name": "Internal Information",
-                "address": settings.SNAPPY_EMAIL}],
-            ticket_id=nonce,
-            scope="private",
-            staff_id=settings.SNAPPY_STAFF_ID
-        )
+    # Add opt-out link
+    optout_url = settings.SITE_DOMAIN_URL + \
+        "/controlinterface/subscription/?msisdn=" + \
+        contact["msisdn"].replace("+", "%2B")
```
Comment (imsickofmaps) [inline snappybouncer/tasks.py]: please use urllib
`import urllib`
`urllib.quote_plus("+27")`

**Comment by imsickofmaps** [comment]
minor fix then :+1: when travis happy

File: snappybouncer/tasks.py (hunk 2 above)
Comment (gsvr) [inline snappybouncer/tasks.py]:
```
import urllib
urllib.quote_plus("+27")
```

**Pull request merged**